src/sentry/api/endpoints/bundle_analysis.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import annotations
+
+from enum import Enum
+
+from rest_framework import serializers
+from rest_framework.permissions import BasePermission
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint, ProjectReleasePermission
+from sentry.apidocs.hooks import HTTP_METHOD_NAME
+from sentry.models.project import Project
+from sentry.sentry_metrics.client import generic_metrics_backend
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
+
+
+class ResourceSizeType(Enum):
+    TOTAL = "total"
+    JAVASCRIPT = "javascript"
+    CSS = "css"
+    FONTS = "fonts"
+    IMAGES = "images"
+
+
+MINUTE = 60  # 60 seconds
+
+
+@region_silo_endpoint
+class BundleAnalysisEndpoint(ProjectEndpoint):
```

Comment:
Thoughts on letting this endpoint take an array of bundle stats? Then the client can make one request to upload everything.

Reply:
👍 Yeah that makes a lot of sense, should be an easy change

Reply:
done!